src/fs-plus.coffee

```diff
@@ -95,7 +96,7 @@ fsPlus =
 
   # Public: Returns true if a file or folder at the specified path exists.
   existsSync: (pathToCheck) ->
-    isPathValid(pathToCheck) and (statSyncNoException(pathToCheck) isnt false)
+    isPathValid(pathToCheck) and (isThere.sync(pathToCheck) isnt false)
```

Comment:
Since the `fs.exists` case does still need to be addressed what do you think about just updating that in this PR to just a `fs.stat` call instead?